app/page.tsx

```
@@ -312,39 +324,52 @@ export default function Home() {
+    const activePath = openTabs[activeTabIndex];
+    const toFetch = [
+      ...new Set([
+        activePath,
+        ...openTabs.filter((path) => !(path in fileContentCache)),
+      ].filter(Boolean)),
+    ].filter((path) => !fetchingPathsRef.current.has(path));
+    if (toFetch.length === 0) return;
+    const aborted = new Set<string>();
+    toFetch.forEach((path) => fetchingPathsRef.current.add(path));
+    toFetch.forEach((path) => {
+      fetch(`/api/file-content?path=${encodeURIComponent(path)}`)
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)